docs/pywbemclicommands.rst

```diff
@@ -387,22 +387,13 @@ Instance associators command
 The ``instance associators`` command lists the CIM instances that are associated
 with the specified source instance.
 
-The source instance can be specified in two ways:
-
-* By specifying an untyped WBEM URI of an instance path in the
-  ``INSTANCENAME`` argument. The namespace of the instance is the namespace
-  specified in the WBEM URI, or otherwise the namespace specified with the
-  ``-namespace``/``-n`` command option, or otherwise the default namespace
-  of the connection. Any host name in the WBEM URI will be ignored.
-  For details, see :ref:`Specifying the INSTANCENAME command argument`.
-
-* By specifying the ``--interactive`` command option and a class name in the
-  ``INSTANCENAME`` argument. The instances of the specified class are displayed
-  and the user is prompted for an index number to select an instance. The
-  namespace of the instance is the namespace specified with the
-  ``-namespace``/``-n`` command option, or otherwise the default namespace
-  of the connection.
-  For details, see :ref:`Interactively selecting INSTANCENAME`.
+The instance name (INSTANCENAME argument) can be specified in two ways as
+defined in :ref:`Interactively selecting INSTANCENAME`:
+
+* By specifying an untyped WBEM_URI.
+
+* By specifying the WBEM_URI with the wild card "?" in place of the keys
+  component of the WBEM_URI, (i.e. <classname>.?).
```

Comment:
Use "CLASSNAME" instead of "<classname>", for consistency.

Reply:
Did not quite understand this one.  Hope I fixed it correctly.

Reply:
The `<classname>` was not showing up in the view mode of my comment. If you go into edit mode, you can see it. Sorry for not checking that.
You guessed correctly in what you fixed.